docs/develop/financial-accounting.md

```diff
@@ -0,0 +1,187 @@
+example, in your personal accounting you would separately track the cash in your physical wallet
+from the funds in your checking account, even though both of those are assets. The bank would split
+out mortgages from car loans, even though both of those are also assets for the bank.
+
+## Double-Entry Bookkeeping
+
+Categorizing accounts into different types is useful for organizational purposes, but it also
+provides a key error-correcting mechanism.
+
+Every record in our accounting is not only recorded in one place, but in two. This is double-entry
+bookkeeping. Why would we do that?
+
+Let's think about the bank loan in our example above. When you took out the loan, two things
+actually happened at the same time. On the one hand, you now owe the bank $100. At the same time,
+the bank gave you $100. These are the two entries that comprise the loan transaction.
+
+From your perspective, your liability to the bank increased by $100 while your assets also increased
+by $100. From the bank's perspective, their assets (the loan to you) increased by $100 while their
+liabilities (the money in your bank account) also increased by $100.
+
+Double-entry bookkeeping ensures that funds are always accounted for. Money never just appears.
+**Funds always go from somewhere to somewhere.**
+
+## Keeping Accounts in Balance
+
+Now we understand that there are different types of accounts and every transaction will be recorded
+in two (or more) accounts -- but which accounts?
+
+The [Fundamental Accounting Equation](https://en.wikipedia.org/wiki/Accounting_equation) stipulates
+that:
+
+**Assets = Liabilities + Equity**
+
+Using our loan example, it's no accident that the loan increases assets and liabilities at the same
+time. Assets and liabilities are on the opposite sides of the equation, and both sides must be
+exactly equal. Loans increase assets and liabilities equally.
+
+Here are some other types of transactions that would affect assets, liabilities, and equity, while
+maintaining this balance:
+
+- If you withdraw $100 in cash from your bank account, your total assets stay the same. Your bank
+  account balance (an asset) would decrease while your physical cash (another asset) would increase.
+- From the perspective of the bank, you withdrawing $100 in cash decreases their assets in the form
+  of the cash they give you, while also decreasing their liabilities because your bank balance
+  decreases as well.
+- If a shareholder invests $1000 in the bank, that increases both the bank's assets and equity.
+
+Assets, liabilities, and equity are types of accounts on the balance sheet, meaning they represent a
+point in time. The other two main categories, revenue and expenses, are income statement accounts,
+meaning they represent flows of money in and out.
+
+Revenue and expenses impact the balance sheet over time. The expanded accounting equation can be
+written as:
+
+**Assets = Liabilities + Contributed Capital + Revenue − Expenses − Dividends**
```

Comment:
Best to keep Equity (rather than reduce Equity to Contributed Capital—it's more than that). I think we can keep Dividends out here to keep things simple.